sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/patch/implementation/PatchUtil.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.patch.implementation;
+
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.patch.PatchOperation;
+import com.azure.cosmos.patch.PatchOperationCore;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import java.util.List;
+
+public class PatchUtil {
+
+    public static String serializePatchOperations(List<PatchOperation<?>> patchOperations, RequestOptions options) {
+        ArrayNode operations =  Utils.getSimpleObjectMapper().createArrayNode();
+
+        for(PatchOperation<?> patchOperation : patchOperations) {
+
+            JsonSerializable jsonSerializable = new JsonSerializable();
+            jsonSerializable.set(PatchConstants.PropertyNames_OperationType, patchOperation.getOperationType().getStringValue());
+
+            if(patchOperation instanceof PatchOperationCore) {
+                jsonSerializable.set(PatchConstants.PropertyNames_Path, ((PatchOperationCore)patchOperation).getPath());
+            }
+
+            jsonSerializable.set(PatchConstants.PropertyNames_Value, patchOperation.getResource());
+            operations.add(jsonSerializable.getPropertyBag());
+        }
+
+        return operations.toString();
```

Comment:
this is what goes over wire, right?
if so it is more optimized to encode to byte[] here. otherwise, you are doing toString here and later to byte[] when written on the wire.

Reply:
Agreed. Now I am converting directly to byte[] and returning the same. Thanks.